clients/java/signalr/src/main/java/HubConnection.java

```diff
@@ -29,9 +29,11 @@ public HubConnection(String url, Transport transport){
             // Adding this to avoid getting error messages on pings for now.
             for (InvocationMessage message : messages) {
                 if (message != null && handlers.containsKey(message.target)) {
-                    ArrayList<Object> types = gson.fromJson((JsonArray)message.arguments[0], (new ArrayList<Object>()).getClass());
-
-                    handlers.get(message.target).invoke(types.toArray());
+                    ArrayList<Object> args = gson.fromJson((JsonArray)message.arguments[0], (new ArrayList<Object>()).getClass());
+                    List<ActionBase> actions = handlers.get(message.target);
```

Comment:
do you need a null check here for the value returned from `get`?

Reply:
Yup, you're right